scripts/simulate.js

```diff
@@ -109,14 +110,27 @@ async function fiatLeverage(amount) {
 
       if (daiEarned.gte(BigNumber.from(0))) {
         totalDaiEarned = totalDaiEarned.add(daiEarned);
+        totalFiatBorrowed = totalFiatBorrowed.add(receipt.fiatDebt);
         totalInterestPaid = totalInterestPaid.add(interestDai);
         daiBalance = receipt.daiBalance;
         gasDai = gasDai.add(receipt.gasDai);
         cycles++;
       }
-    }
+      else {
```

Comment:
because I only want it to run once, right before the loop ends. the if condition just happens to check for the loop continuing (to avoid adding an unprofitable loop to the count) so an else on that condition gives exactly what I want: execution only on the last loop